Doc/library/shutil.rst

```diff
@@ -67,6 +67,10 @@ Directory and files operations
    a new symbolic link will be created instead of copying the
    file *src* points to.
 
+   *allow_reflink* enables copy-on-write on supported Linux filesystems.
+   The :c:func:`copy_file_range` system call is used internally when
```

Comment:
The implementation uses the Python implementation, so you should drop the ``:c:`` prefix:

```suggestion
   The :func:`copy_file_range` system call is used internally when
```

Reply:
Oh, I missed the fact that you mention "system call". Since the implementation uses the Python function call, I suggest to point to the Python function. Curious readers can follow the link to see how the Python function is implemented.